locust/env.py

```diff
@@ -65,10 +60,64 @@ def  __init__(
         else:
             self.events = Events()
         
-        self.options = options
+        self.locust_classes = locust_classes
+        self.stats = RequestStats()
         self.host = host
         self.reset_stats = reset_stats
         self.step_load = step_load
         self.stop_timeout = stop_timeout
         self.catch_exceptions = catch_exceptions
+    
+    def _create_runner(self, runner_class, *args, **kwargs):
```

Comment:
> Hmm... the method creates a new runner and then sets and returns it. Wouldnt it be more clear if it only set it?

:+1: I've updated the documentation and examples to access `runner` and `web_ui` instance through the `Environment` instance. However, I didn't remove the returning of the instance as I don't see much harm in it, and this way we don't have to update 30+ tests that relies on it (in many places like this: `runner = Environment(locust_classes=[L1, L2, L3]).create_local_runner()`).

> couldnt we just expose this method publicly? the wrappers dont really seem to do much..

Having the the `create_xxx_runner` methods provides a nice API for end-users who wants to use Loucst as a lib. All of those methods take different arguments and having them removes the need for people to know about the different runner classes. It also allows us to include those methods in the API docs in a simple way, as well as API docs for the base `LocustRunner` which can show people the common runner API.

Reply:
Ok! To me env.create_runner(LocalRunner) is clearer than env.create_local_runner(), or at least about the same level of clarity and doesnt require wrapper methods. But it is not a big thing for me.

Reply:
:+1: 

(When just looking at the LocalRunner I agree, but since the Master and Worker runners requires other arguments, I don't think this would be a nice API `env.create_runner(runner_class, *args, **kwargs)` (and hard to document in a good way))

Reply:
Yes, that make sense!